Conversions/DecimalToBinary.js

```diff
@@ -1,13 +1,35 @@
-function decimalToBinary (num) {
-  const bin = []
-  while (num > 0) {
-    bin.unshift(num % 2)
-    num >>= 1 // basically /= 2 without remainder if any
-  }
-  return bin.join('')
+function decimalToBinary(num) {
+    const bin = []
+    while (num > 0) {
+        bin.unshift(num % 2)
+        num >>= 1 // basically /= 2 without remainder if any
+    }
+    return bin.join('')
 }
 
-export { decimalToBinary }
+// using .toString() method
+
+function decimalToBinary_2(dec) {
+    return dec.toString(2)
+}
+
+// using Remainder Quotient method
+
+function decimalToBinary_3(dec) {
+    let bin = 0;
+    let rem, i = 1,
+        step = 1;
+    while (dec != 0) {
+        rem = dec % 2;
+        dec = parseInt(dec / 2);
```

Comment:
Looks like a hacky reimplementation of `decimalToBinary` to me, except instead of using an array to store the digits this uses a base 10 number which may then be stringified in base 10 to produce the desired binary number.

The `parseInt` hack to achieve floor division rather than using `dec >>= 1` is a clear downgrade. You could consider `Math.floor(dec / 2)` for readability, but relying on `parseInt` to perform flooring is hacky IMO.